src/functionsDelete.js

```diff
@@ -50,8 +51,9 @@ var printTooManyOps = function(projectId) {
 
 module.exports = function(functionsToDelete, projectId, appEngineLocation) {
   deletes = _.map(functionsToDelete, function(name) {
-    var scheduleName = helper.getScheduleName(name, appEngineLocation);
-    var topicName = helper.getTopicName(name);
+    const scheduleName = helper.getScheduleName(name, appEngineLocation);
+    const topicName = helper.getTopicName(name);
+    const functionName = helper.getFunctionName(name);
```

Comment:
nice catch. can we update line 89 with this as well?

Reply:
done.